pkgs/development/python-modules/bugzilla/default.nix

```diff
@@ -11,7 +17,7 @@ buildPythonPackage rec {
     sha256 = "0q8c3k0kdnd11g2s56cp8va9365x0xfr2m2zn9fgxjijdyhwdic5";
   };
 
-  buildInputs = [ pep8 coverage logilab_common ];
+  buildInputs = [ pep8 coverage logilab-common ];
```

Comment:
```suggestion
  buildInputs = [ logilab-common ];
```
no linters, please

Reply:
Done! It turns out logilab-common is no longer needed either.